Essentials/src/main/java/com/earth2me/essentials/commands/Commandcondense.java

```diff
@@ -161,8 +164,7 @@ private SimpleRecipe getCondenseType(final ItemStack stack) {
                 iter.remove();
                 continue;
             }
-
-            if (inputSlot.getDurability() == Short.MAX_VALUE) {
+            if (legacy && inputSlot.getDurability() == Short.MAX_VALUE) {
```

Comment:
Why create a variable when it's only used once? You should do this in `Trade.java` instead

Reply:
@l1ttleO It is far more readable with a variable than with inlined version checks everywhere.

In theory this could be a static field, but I'll leave it to @JRoy to make that change if he decides to.

Reply:
Because otherwise we'd be repeating the same check in a loop which isn't cheap